apps/web/src/views/Farms/utils/getMerklLink.tsx

```diff
@@ -0,0 +1,24 @@
+import { ChainId } from '@pancakeswap/chains'
+import { MERKL_POOLS } from 'config/merkl'
+import { Address } from 'viem'
+
+export const getMerklLink = ({
+  chainId,
+  lpAddress,
+}: {
+  chainId?: ChainId | number
+  lpAddress?: Address
+}): string | false => {
```

Comment:
Can we unified the return type here? As returning string | false doesn't quite match the meaning of the get function
```suggestion
}): string | undefined => {
```